src/ape_plugins/_cli.py

```diff
@@ -168,7 +169,62 @@ def install(cli_ctx, skip_confirmation):
             # NOTE: Be *extremely careful* with this command, as it modifies the user's
             #       installed packages, to potentially catastrophic results
             # NOTE: This is not abstracted into another function *on purpose*
-            subprocess.call([sys.executable, "-m", "pip", "install", "--quiet", f"{package_name}"])
+            result = subprocess.call(
+                [sys.executable, "-m", "pip", "install", "--quiet", f"{package_name}"]
+            )
+            if result != 0:
+                cli_ctx.logger.error(f"Failed to add '{package_name}'.")
+                any_install_failed = True
+    if any_install_failed:
+        sys.exit(1)
+
+
+@cli.command(short_help="Uninstall all plugins in the local config file")
+@ape_cli_context()
+@skip_confirmation_option("Don't ask for confirmation to install the plugins")
+def uninstall(cli_ctx, skip_confirmation):
+    any_uninstall_failed = False
+    plugins = config.get_config("plugins") or []
+    for plugin in plugins:
+        # name schema is wrong
+        if plugin.startswith("ape"):
+            cli_ctx.abort(
+                f"Namespace 'ape' in '{plugin}' is not required. Please remove the pre-fix ape."
+            )
+            any_uninstall_failed = True
+
+        module_name, package_name = extract_module_and_package_install_names(plugin)
+        # check for installed check the config.yaml
+        if not is_plugin_installed(module_name):
+            cli_ctx.logger.warning(f"Plugin '{plugin}' is not installed.")
+            any_uninstall_failed = True
+
+        # if plugin is installed but not a 2nd class. It must be a third party
+        if module_name not in github_client.available_plugins and is_plugin_installed(plugin):
+            cli_ctx.logger.warning(
+                f"Plugin '{module_name}' is installed but not in available plugins."
+                f" Please uninstall outside of Ape."
+            )
+            any_uninstall_failed = True
+
+        # if plugin is installed and 2nd class. We should uninstall it
+        if is_plugin_installed(module_name) and (
```

Comment:
We are calling `is_plugin_installed` twice.
We are also checking if the `module_name` is in the available plugins list twice.

Can we just store the result of those checks as `boolean` variables and re-use them?